Formula/prokka.rb

```diff
@@ -0,0 +1,49 @@
+class Prokka < Formula
+  # cite Seemann_2014: "https://doi.org/10.1093/bioinformatics/btu153"
+  desc "Rapid annotation of prokaryotic genomes"
+  homepage "https://github.com/tseemann/prokka"
+  url "https://github.com/tseemann/prokka/archive/v1.13.tar.gz"
+  sha256 "0090acd6e3a8cb2d517fe0933c717c4f8271c60d0a493f7c4927504d77a8db4f"
+
+  depends_on "cpanminus" => :build
+
+  depends_on "aragorn"
+  depends_on "barrnap"
+  depends_on "bioperl"
+  depends_on "blast"
+  depends_on "hmmer"
+  depends_on "infernal"
+  depends_on "less" unless OS.mac?
+  depends_on "minced"
+  depends_on "parallel"
+  depends_on "perl" unless OS.mac?
+  depends_on "prodigal"
+  depends_on "tbl2asn"
```

Comment:
I like to group the macOS dependencies at the end.
```ruby
unless OS.mac?
  depends_on "less"
  depends_on "perl"
end
```

Reply:
```ruby
depends_on :java
```
?

Reply:
`minced` already depends on java, so i don't need to do it explicitly. it's the only use for it.